helpers/storagedriver.py

```diff
@@ -13,8 +13,18 @@
 #
 # Open vStorage is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY of any kind.
+import json
+
+try:
+    from ovs.extensions.services.service import ServiceManager
```

Comment:
The current master branch points towards FWK 2.11
No need to have this try-except stuff